video/out/vulkan/context.c

```diff
@@ -29,12 +29,31 @@ struct vulkan_opts {
     int async_compute;
 };
 
+static VkPhysicalDevice *vk_get_physical_devices(VkInstance inst, uint32_t *num_devices) {
+    VkResult res;
+    VkPhysicalDevice *devices = NULL;
+
+    res = vkEnumeratePhysicalDevices(inst, num_devices, NULL);
+    if (res != VK_SUCCESS)
+        goto done;
+
+    devices = talloc_array(NULL, VkPhysicalDevice, *num_devices);
+    vkEnumeratePhysicalDevices(inst, num_devices, devices);
+    if (res != VK_SUCCESS) {
```

Comment:
This branch is unreachable - you probably meant to do this instead:

```suggestion
    res = vkEnumeratePhysicalDevices(inst, num_devices, devices);
    if (res != VK_SUCCESS) {
```